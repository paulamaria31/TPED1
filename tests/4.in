100
XOVVOXOVV
XOXXXOOVV
XVVXXVXVO
VVXOXVOXV
VVOOXXXOV
XOVXVXVXO
VVVXVXXVV
XVOVVVOVV
XOOXXOOXX
VXXOOOOVV
OXXVOVXOX
XOVOOXVXX
OVVXVXOVX
OXVVOOVXV
XXXOXOXXX
VXOOOVOXV
OVOOOOVXV
OVXXOVOOO
XXXVVVVVX
VOXVXVVOV
OOXOXVVVV
VOVVXOXXX
XXOXOXXVX
XVVOVOXXV
VOXVXVOXX
VVOXVOOXV
VVXOVVXXV
XOXOOXXXO
OOXOVOXXO
OXOOVVVVV
OXVVOVOOO
XXVXXXOVO
VOXVXOOVO
VOXVOVXXO
OOOXXOOVX
XVVVVVXOX
OVXVOXVOV
OXOXXVVXV
XXXXVXOVX
OXXOXVVOO
OVVOVVOOO
VOXOOOOXX
OVOVOVOXO
OVXXOOXOX
XOOOXVXVV
VVXVXXXOX
OVVXVOVXO
OXOVOVXVX
VXVOXXOOX
OXVOOVXOX
VOOVOXVOO
XXOVVOOVV
OVOVVXVOX
VOVVOXVVX
OVOOVXXOO
VVXVXOVVX
XVVVOVXXO
VXVXVOOOX
VVXVXOVXO
VVVOXOXXO
XOOOOXXVO
XVVVXXOXO
VOOVOOXXV
VOVXOOVOX
XXXOXVOVV
VOXOOOXVO
OVOOXVXXX
XOVOXVOXX
XXVVVOVOO
VXVOOXOOV
VOVVVXVOO
VXOOOVXOX
XOOVXOVOV
XOOXXXVVX
VXVXVOXVV
XXOVVVVXV
VOOXOOVOV
VXVOXOXXX
VVOXVOOXV
VVVXXXOVO
OXXXXVXOO
OOXVVXOOX
OOOOOOOOO
OOVOVVXVO
VOVXOOVVX
OOXOOXOXV
VOVXVXOXO
VVXXXXOVO
OXVXOVVXX
OVVXVVVOX
XOVXVOVOV
VVXVXXXVO
OXXXOOXXX
XOVVVOOXV
VXOOVVXOV
VVVOXXXXO
OVOXOXVXO
XXXVOOVVO
XOOOVVVOX
XOXVXOVXX
