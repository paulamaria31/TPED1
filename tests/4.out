Tabuleiro 1 em andamento [X: sem jogada mestre]
Tabuleiro 2 em andamento [O: sem jogada mestre]
Tabuleiro 3 invalido
Tabuleiro 4 em andamento [O: (1,1)]
Tabuleiro 5 em andamento [proximo jogador indefinido]
Tabuleiro 6 invalido
Tabuleiro 7 invalido
Tabuleiro 8 em andamento [X: sem jogada mestre]
Tabuleiro 9 com vitoria [X]
Tabuleiro 10 invalido
Tabuleiro 11 em andamento [O: sem jogada mestre]
Tabuleiro 12 em andamento [O: sem jogada mestre]
Tabuleiro 13 em andamento [O: sem jogada mestre]
Tabuleiro 14 em andamento [X: sem jogada mestre]
Tabuleiro 15 invalido
Tabuleiro 16 invalido
Tabuleiro 17 invalido
Tabuleiro 18 invalido
Tabuleiro 19 invalido
Tabuleiro 20 em andamento [proximo jogador indefinido]
Tabuleiro 21 em andamento [X: (3,1)]
Tabuleiro 22 invalido
Tabuleiro 23 invalido
Tabuleiro 24 em andamento [O: (2,2)]
Tabuleiro 25 invalido
Tabuleiro 26 em andamento [X: sem jogada mestre]
Tabuleiro 27 invalido
Tabuleiro 28 deu velha
Tabuleiro 29 invalido
Tabuleiro 30 invalido
Tabuleiro 31 invalido
Tabuleiro 32 invalido
Tabuleiro 33 invalido
Tabuleiro 34 em andamento [proximo jogador indefinido]
Tabuleiro 35 invalido
Tabuleiro 36 invalido
Tabuleiro 37 em andamento [X: (3,3)]
Tabuleiro 38 invalido
Tabuleiro 39 invalido
Tabuleiro 40 em andamento [X: (3,1)]
Tabuleiro 41 invalido
Tabuleiro 42 invalido
Tabuleiro 43 invalido
Tabuleiro 44 em andamento [proximo jogador indefinido]
Tabuleiro 45 em andamento [proximo jogador indefinido]
Tabuleiro 46 invalido
Tabuleiro 47 em andamento [X: sem jogada mestre]
Tabuleiro 48 em andamento [proximo jogador indefinido]
Tabuleiro 49 em andamento [O: (1,1)]
Tabuleiro 50 em andamento [X: sem jogada mestre]
Tabuleiro 51 invalido
Tabuleiro 52 em andamento [X: sem jogada mestre]
Tabuleiro 53 em andamento [X: sem jogada mestre]
Tabuleiro 54 em andamento [proximo jogador indefinido]
Tabuleiro 55 invalido
Tabuleiro 56 invalido
Tabuleiro 57 em andamento [O: sem jogada mestre]
Tabuleiro 58 em andamento [proximo jogador indefinido]
Tabuleiro 59 em andamento [O: sem jogada mestre]
Tabuleiro 60 em andamento [proximo jogador indefinido]
Tabuleiro 61 invalido
Tabuleiro 62 invalido
Tabuleiro 63 invalido
Tabuleiro 64 invalido
Tabuleiro 65 invalido
Tabuleiro 66 invalido
Tabuleiro 67 com vitoria [X]
Tabuleiro 68 com vitoria [X]
Tabuleiro 69 em andamento [X: (1,3)]
Tabuleiro 70 invalido
Tabuleiro 71 invalido
Tabuleiro 72 em andamento [X: sem jogada mestre]
Tabuleiro 73 invalido
Tabuleiro 74 invalido
Tabuleiro 75 invalido
Tabuleiro 76 invalido
Tabuleiro 77 invalido
Tabuleiro 78 invalido
Tabuleiro 79 em andamento [X: sem jogada mestre]
Tabuleiro 80 com vitoria [X]
Tabuleiro 81 invalido
Tabuleiro 82 com vitoria [X]
Tabuleiro 83 invalido
Tabuleiro 84 invalido
Tabuleiro 85 em andamento [X: sem jogada mestre]
Tabuleiro 86 invalido
Tabuleiro 87 em andamento [proximo jogador indefinido]
Tabuleiro 88 invalido
Tabuleiro 89 invalido
Tabuleiro 90 em andamento [proximo jogador indefinido]
Tabuleiro 91 em andamento [X: (3,1)]
Tabuleiro 92 invalido
Tabuleiro 93 invalido
Tabuleiro 94 em andamento [X: sem jogada mestre]
Tabuleiro 95 em andamento [X: sem jogada mestre]
Tabuleiro 96 invalido
Tabuleiro 97 com vitoria [O]
Tabuleiro 98 com vitoria [X]
Tabuleiro 99 invalido
Tabuleiro 100 invalido
